### DISCUSSION TITLE
Update SimpleLogin server location in website's FAQ

### DISCUSSION THREAD

**Discussion by Vnzq4Qje3zzU** [simple-login/app#942] [Feature Requests]
I think it's now using ProtonMail's servers? It's also interesting to give more an overview about how much of the infrastructure is shared.

Another thing is the footer "SimpleLogin is the product of SimpleLogin SAS, registered in France under the SIREN number 884302134", which I think it's better to include a notice about ProtonMail.

**Comment by nguyenkims** [comment]
@bx2KdaukwF the privacy page has been updated to include the new Proton servers https://simplelogin.io/privacy/

> Another thing is the footer "SimpleLogin is the product of SimpleLogin SAS, registered in France under the SIREN number 884302134", which I think it's better to include a notice about ProtonMail.

SimpleLogin will continue to operate as an independent service and SimpleLogin SAS will continue to be based in Paris, France.
